step.yml

```diff
@@ -1,4 +1,4 @@
-title: "Flutter Test"
```

Comment:
Just out of curiosity: why did we remove the quotations in this file? I know YAML doesn't necessarily need quotes for strings, but we use them in almost all `step.yml`'s I've seen.

Reply:
I like the quotation-less version, so I remove it wherever it makes sense. We don't have a YAML style guide (yet)

Reply:
Although we don't at the moment, I think we should strive for some level of consistency. Therefore, I'd suggest not removing these now. Wdyt?